apps/portals/cancercomplexity/src/pages/CCKPSearchPage.tsx

```diff
@@ -94,14 +96,28 @@ export function CCKPSearchPage(props: CCKPSearchPageProps) {
         searchPageTabs[tabIndex].count = newCount
         setSearchPageTabsState([...searchPageTabs])
       }
+      // PORTALS-3382: If no tab is selected and all counts have been set, then redirect to the item with the highest count
+      const allCountsSet = searchPageTabs.every(tab => tab.count !== undefined)
+      if (selectedTabIndex == undefined && allCountsSet) {
+        // navigate to the tab that has the highest count
+        const maxCountTab = searchPageTabs.reduce((max, tab) =>
+          max === null || tab.count! > max.count! ? tab : max,
```

Comment:
Is this too dense?

Reply:
I think it's ok. I might suggest splitting the null case from the comparison with the accumulator. The tricky thing about this statement is trying to remember order of operations between a ternary, `||`, `>`, and `===`.

That said, I don't know that you need a null check here. The initial value will be `searchPageTabs[0]`, so `max` should never be `null`.

So I'd either separate the null check into its own statement or remove it altogether